src/org/way2it/data_structures/binary_tree/BinaryTree.java

```diff
@@ -16,19 +16,71 @@ public BinaryTree() {
     // Values that are greater than the value in the current node - should be placed in the right subtree
     public void add(int value) {
         // TODO implement me
+        Node addNode = new Node();
+        addNode.value = value;
+        addNode.left = null;
+        addNode.right = null;
+        if (root == null) {
+            root = addNode;
+        } else {
+            Node thisNode = root;
+            while (thisNode.right != null && thisNode.left != null) {
+                if (value > thisNode.value) thisNode = thisNode.right;
+                if (value <= thisNode.value) thisNode = thisNode.left;
```

Comment:
Use curly brackets for one-line if, please 
```suggestion
                if (value > thisNode.value) { 
                thisNode = thisNode.right; 
                }
                if (value <= thisNode.value) { 
                thisNode = thisNode.left;
                }
```